PyFunceble/cli/__init__.py

```diff
@@ -344,8 +345,9 @@ def tool():  # pragma: no cover pylint: disable=too-many-branches,too-many-state
                     "-f",
                     "--file",
                     type=str,
-                    help="Read the given file and test all domains inside it. "
-                    "If a URL is given we download and test the content of the given URL.",  # pylint: disable=line-too-long
+                    help="Read a local or remote (RAW link) file and test all domains inside it."
+                    "\nIf remote (RAW link) file is given, PyFunceble will download it, "
+                    "and test the content of the given RAW link as if it was a locally stored file.",
```

Comment:
Self note:

`RAW Link` can lead to misunderstanding between domain test `-f` vs link test with `-uf`

The `RAW link` is used to reference the raw.guthubusercontent.bla/bla/bla

... need more thoughts